crypto.go

```diff
@@ -432,8 +461,14 @@ func (s x25519Scheme) DeserializePrivate(enc []byte) (KEMPrivateKey, error) {
 		return nil, fmt.Errorf("Error deserializing X25519 private key")
 	}
 
+	copied := clampX25519(enc)
+	var zero [32]byte
+	if subtle.ConstantTimeCompare(copied, zero[:]) == 1 {
```

Comment:
This is insufficient to check for zero keys. You need to reduce modulo the curve order

Reply:
Yeah, since it's a SHOULD, I removed that check.